.styleci.yml

```diff
@@ -0,0 +1,7 @@
+preset: psr2
+
+risky: false
+
+finder:
+  name:
```

Comment:
This does not exclude vendor directories. you really don't want this hitting packages.

Reply:
Does StyleCI run composer install prior to running its analysis? We have vendor gitignored so this seems like it would be a nice bit of insurance against accidentally checking in vendor.

Reply:
Either way you shouldn't include the file until we decide what is going in it. And this is ancillary to the bug fix.

Reply:
Rebased it out.